src/ExerciseRunner/CgiRunner.php

```diff
@@ -43,6 +43,11 @@ class CgiRunner implements ExerciseRunnerInterface
      */
     public function __construct(CgiExercise $exercise, EventDispatcher $eventDispatcher)
     {
+        // Suppress shell output. Anything non-zero is a failure.
+        @system('php-cgi --version > /dev/null 2>&1', $failedToRun);
+        if ($failedToRun) {
+            die('Could not load php-cgi binary. Please install php-cgi using your package manager.' . PHP_EOL);
```

Comment:
I think I would prefer if we threw an exception here? Maybe `RuntimeException`, we would then have to add an extra `catch` in `Application::run`, just so there is one entry and exit point for the app, what do you think?